app/uk/gov/hmrc/helptosavefrontend/connectors/NSIConnector.scala

```diff
@@ -130,6 +131,18 @@ class NSIConnectorImpl @Inject() (conf: Configuration, metrics: Metrics) extends
       }
   }
 
+  override def healthCheck(userInfo: NSIUserInfo)(implicit hc: HeaderCarrier, ex: ExecutionContext): Result[Unit] = EitherT[Future, String, Unit]{
+    httpProxy.put(nsiHealthCheckUrl, userInfo, Map(nsiAuthHeaderKey → nsiBasicAuth))(nsiUserInfoFormat, hc.copy(authorization = None), ex)
+      .map[Either[String, Unit]] { response ⇒
+        response.status match {
+          case Status.OK ⇒ Right(())
+          case other     ⇒ Left(s"Received unexpected status $other from NS&I while trying to update email. Body was ${response.body}")
```

Comment:
Is this text correct? Maybe...
HealthCheck received an unexpected http status code $other from NS&I. Body was blah, blah

Reply:
That `String` will eventually be picked up and logged in this line here in `HeathCheck`:

https://github.com/hmrc/help-to-save-frontend/blob/master/app/uk/gov/hmrc/helptosavefrontend/health/HealthCheck.scala#L110